db/migrate/20210809095635_add_token_to_user_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddTokenToUserNotifications < ActiveRecord::Migration
+  def change
+    add_column :user_notifications, :token, :string
```

Comment:
Might be useful to add an index here. https://medium.com/@mera.stackhouse/what-are-indexes-and-how-to-add-them-to-your-rails-app-dc066d538771